### PR TITLE
fix: connection manager

### PR DESCRIPTION
**Pull request by vitaligi**
* fix connection manager initialization and DI
* fix resources cleanup on termination
* fix type definition

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: src/common/connectionManager.ts
Comment (CL-SHLOMIKONCHA) [inline src/common/connectionManager.ts:1]: Please change to our log convension ({msg: 'bla bla'}), exmaple:
image

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]
Please check the case of reconnection

File: src/common/connectionManager.ts
Comment (vitaligi) [reply src/common/connectionManager.ts:1]: done

**Comment by vitaligi** [comment]
> Please check the case of reconnection

checked manually TypeORM is able to reconnect

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Pull request merged**